CHANGELOG.md

```diff
@@ -11,13 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Removes stringify call on python constructor parameters. https://github.com/microsoftgraph/msgraph-sdk-python/issues/165
+
 ### Changed
 
-- Fixed a stack overflow in the core generator caused by circular comparisons. [#5369](https://github.com/microsoft/kiota/issues/5369)
-- Fixed a bug where collection/array of primitive types members for union/intersection types would be ignored. [#5283](https://github.com/microsoft/kiota/issues/5283)
-- Fixed a when generating a plugin when only an operation is selected in the root node in the extension. [#5300](https://github.com/microsoft/kiota/issues/5300)
-- Fixed a bug where function descriptions in plugin manifest defaults to path summary instead of description[#5301](https://github.com/microsoft/kiota/issues/5301)
-- Fixed a bug where the description special characters are encoded. [5286](https://github.com/microsoft/kiota/issues/5286)
+- Fixed a bug where python construcor parameters are beig cast to strings leading to bugs as the types is unkown on graph call.
```

Comment:
Any chance you can double check the deleted entries in the changelog here?

Reply:
I just rebased and force pushed here to clean things up...